xknxproject/loader/project_loader.py

```diff
@@ -277,10 +279,31 @@ def parse_space(self, node: ElementTree.Element) -> XMLSpace:
             elif sub_node.tag.endswith("DeviceInstanceRef"):
                 if individual_address := self.devices.get(sub_node.get("RefId", "")):
                     space.devices.append(individual_address)
+            elif sub_node.tag.endswith("Function"):
+                space.functions.append(self.parse_functions(sub_node))
 
         return space
 
 
+    def parse_functions(self, node: ElementTree.Element) -> XMLFunctions:
+
+        project_uid = node.get("Puid")
+        functions: XMLFunctions = XMLFunctions (
+                    identifier=node.get("Id"),  # type: ignore[arg-type]
+                    name=node.get("Name"),  # type: ignore[arg-type]
+                    function_type="",
+                    project_uid=int(project_uid) if project_uid else None,
+                    group_addresses=[],
+        )
+
+        for sub_node in node:
+            if sub_node.tag.endswith("GroupAddressRef"):
+                refId=sub_node.get("RefId", "")
+                functions.group_addresses.append(refId)
```

Comment:
consider using a list comprehension 
```suggestion
                    group_addresses=[sub_node.get("RefId", "") for sub_node in node if sub_node.tag.endswith("GroupAddressRef")],
        )
```